freedom-e-sdk/bsp/env/start.S

```diff
@@ -14,28 +15,33 @@ _start:
 	.cfi_undefined ra
 .option push
 .option norelax
-	//la gp, __global_pointer$
-	lui gp, %hi(__global_pointer$)
-	addi gp, gp, %lo(__global_pointer$)
+1:
+   //la gp, _global_pointer$
+   auipc gp, %pcrel_hi(__global_pointer$)
+	addi gp, gp, %pcrel_lo(1b)
```

Comment:
Remove indentation (maybe github is doing this)

Reply:
Mixed tabs and spaces...